ARMTemplates/201-dtl-create-lab-with-policies/azuredeploy.parameters.json

```diff
@@ -8,11 +8,14 @@
     "labVmShutDownTime": {
       "value": "21:00"
     },
+    "timeZoneId": {
+      "value": "UTC"
+    },
     "maxAllowedVmsPerUser": {
```

Comment:
good catch, on other day I thought customer has put these defaults but it is us who put it other way